### PR TITLE
Integrate updated React SDK

### PR DESCRIPTION
**Pull request by rygine**
This PR integrates the updated React SDK to bring local-first architecture to Inbox web.

### CODE REVIEW

File: cypress/e2e/connect_disconnect.cy.ts [hunk 1]
```diff
@@ -23,25 +23,13 @@ describe(
...
-        "new-message-icon-cta",
-        "conversations-list-panel",
       ];
 
       elements.forEach((element) => {
         checkElement(element);
       });
     });
 
-    it("Shows feedback convo with a preview message", () => {
```
Comment (neekolas) [inline cypress/e2e/connect_disconnect.cy.ts:36]: Did we remove the feedback convo, or did we just break this test with the upgrade?

Comment (rygine) [reply cypress/e2e/connect_disconnect.cy.ts]: it was briefly removed. it's back now!

File: cypress/test_utils.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ const enterWalletAddress = (testUser: string) => {
 
 const checkExpectedPreMessageFields = () => {
   cy.wait(1000);
-  checkElement("message-beginning-text");
+  // checkElement("message-beginning-text");
```
Comment (daria-github) [inline cypress/test_utils.ts:31]: why comment this out? want to just remove this if not needed?

File: src/component-library/components/MessageInput/MessageInput.tsx [hunk 3]
```diff
@@ -153,6 +176,57 @@ export const MessageInput = ({
     status,
   });
 
+  const send = useCallback(async () => {
+    if (value || attachment) {
+      // save reference to these values before clearing them from state
```
Comment (daria-github) [inline src/component-library/components/MessageInput/MessageInput.tsx:181]: thanks for adding the comment!

File: cypress/test_utils.ts (hunk 2 above)
Comment (rygine) [reply cypress/test_utils.ts:31]: oops, yea, i meant to remove that line.

**Review by daria-github — APPROVED** [review]
Code looks good and I did some spot-check testing as well since it's a big refactor. Great job on this. 👏

**Pull request merged**